src/controllers/user.py

```diff
@@ -22,14 +22,17 @@ def create_user(body):
     return result, code
 
 
-def get_all_users():
+def get_all_users(filter=None):
+    if filter:
+        result, code = db.get_all(User, {"invalid": 5})
+        return result, code
+
```

Comment:
this is not necessary as we do not have a user filter.  don't need to test features that we don't have